zoo-css-challenge/index.html

```diff
@@ -123,11 +131,15 @@ <h4>Youth and young adults</h4>
 			<h3>Useful Links</h3>
 			<p>
 			</p>
-			<a href="https://www.w3schools.com">w3schools.com</a>
+			<a href="https://www.w3schools.com" target="_blank">w3schools.com</a>
 			<a href="https://www.nationalgeographic.com/" target="_blank">National Geographic</a>
-			<a href="http://www.wikipedia.org" target="_top">Wikipedia</a>
+			<a href="http://www.wikipedia.org" target="_blank">Wikipedia</a>
+		</section>
+		<section class="myFavAnimal">
+			<h4>Capybaras - who said rodents can't be cute?</h4>
```

Comment:
```suggestion
			<h2>Capybaras - who said rodents can't be cute?</h2>
```
The other animal sections have `h2` so it might make sense to have this one at the same level. Heading levels can be thought of like a table of contents in a word document